src/tools/kvstore_tool.h

```diff
@@ -10,10 +10,18 @@
 #include "acconfig.h"
 #include "include/buffer_fwd.h"
 #ifdef WITH_BLUESTORE
+// kv/KeyValueDB.h is included via os/ObjectMap.h -> os/ObjectStore.h
 #include "os/bluestore/BlueStore.h"
+#else
+#include "kv/KeyValueDB.h"
 #endif
 
-class KeyValueDB;
+#include "global/global_context.h"
+#include "global/global_init.h"
+
```

Comment:
Can't remember if we've discussed this, but - the solution is to have the body of the dtor in the .cc file, not to remove the fwd decl

Reply:
I'm not sure I'm following your suggestion. There is no destructor defined in this header file. The unique_ptr declaration is pulling this string. 

In any case, there is another reason for removing the forward declaration, which I have mentioned in a new comment at line 13:
```
// kv/KeyValueDB.h is included via os/ObjectMap.h -> os/ObjectStore.h
#include "os/bluestore/BlueStore.h"
```
So, the forward declaration line was confusing since it wasn't having any effect but on its own it wouldn't be sufficient. It became obvious when I tried compiling without the BlueStore.

Reply:
OK - that's a good enough reason